lib/testers/runtime-tester.ts

```diff
@@ -0,0 +1,72 @@
+import Logger from "../logger";
+import BaseTester from "./base-tester";
+import child_process from "child_process";
+import Course from "../models/course";
+import Language from "../models/language";
+import util from "util";
+import tmp from "tmp";
+import Dockerfile from "../models/dockerfile";
+
+const exec = util.promisify(child_process.exec);
+
+export default class RuntimeTester extends BaseTester {
```

Comment:
```suggestion
export default class CommandTester extends BaseTester {
```

(Not super sure about this, just leaving as a suggestion)